tools/i18n/makepot.php

```diff
@@ -9,11 +9,11 @@
 
 /**
  * Class to create POT files for
- *  - WordPress 3.4+
- *  - WordPress plugins
- *  - WordPress themes
+ *  - ClassicPress 3.4+
+ *  - ClassicPress plugins
+ *  - ClassicPress themes
  *  - GlotPress (standalone)
- *  - WordPress.org projects (Rosetta, forums, directories)
+ *  - ClassicPress.org projects (Rosetta, forums, directories)
```

Comment:
There is no ClassicPress.org website, should either become our new site or keep as is for now, since we also don't have "(Rosetta, forums, directories)"?